tests/bluetooth/controller/common/include/helper_util.h

```diff
@@ -25,13 +25,13 @@ uint16_t event_counter(struct ll_conn *conn);
 	ut_rx_node_real(__FILE__, __LINE__, _opcode, _ntf_ref, _param)
 #define ut_rx_q_is_empty() ut_rx_q_is_empty_real(__FILE__, __LINE__)
 
-void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```

Comment:
line is now over 100chars
```suggestion
void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```